src/views/ProfileView.vue

```diff
@@ -228,6 +228,11 @@
 
   const twoFactorAuthError: ComputedRef<string> = computed(() => {
     if (twoFactorAuthenticationStore.loading) return '';
+    if (
+      twoFactorAuthenticationStore.errorCode === 'SOFTWARE_TOKEN_VERIFICATION_ERROR' ||
+      twoFactorAuthenticationStore.errorCode === 'OTP_NICHT_GUELTIG'
+    )
+      return '';
```

Comment:
Habe approved aber bitte die Computed hier schönen gestalten um den empty string nicht jedes Mal zu returnen.

```suggestion
const twoFactorAuthError: ComputedRef<string> = computed(() => {
  // Early return if loading
  if (twoFactorAuthenticationStore.loading) return '';

  const ignoredErrorCodes = ['SOFTWARE_TOKEN_VERIFICATION_ERROR', 'OTP_NICHT_GUELTIG'];

  // If the errorCode exists but isn't in the ignoredErrorCodes, return localized error
  if (twoFactorAuthenticationStore.errorCode && !ignoredErrorCodes.includes(twoFactorAuthenticationStore.errorCode)) {
    return t('admin.person.twoFactorAuthentication.errors.connection');
  }

  // Default return, no error
  return '';
});

```